### PR TITLE
ATS-460 : ATS: T-Engines - Update license information to *not* refer to Enterprise edition

### PR DESCRIPTION
**Pull request by tzclucian**
1. Dockerfiles ✅ - remove line related to "Enterprise Edition"
2. LICENCES.md ✅ - remove line related to "Enterprise Edition"
3. Controller Log messages ✅ - remove "Enterprise Edition"
4. LGPL V3 ✅

### CODE REVIEW

**Comment by montgolfiere** [comment]
In terms of red build, it looks as if we may now need to resolve ATS-481 first (and then merge into this branch) ?

File: alfresco-docker-imagemagick/Dockerfile [hunk 1]
```diff
@@ -1,6 +1,5 @@
 # Image provides a container in which to run ImageMagick transformations for Alfresco Enterprise Content Services.
```
Comment (montgolfiere) [inline alfresco-docker-imagemagick/Dockerfile]: Remove "Enterprise" ? Similarly for other T-Engine Dockerfiles.

File: alfresco-transformer-base/src/main/java/org/alfresco/transformer/logging/StandardMessages.java [hunk 2]
```diff
@@ -28,10 +28,7 @@
...
 public interface StandardMessages
 {
-    String ENTERPRISE_LICENCE = 
-        "This image is only intended to be used with the Alfresco Enterprise Content Repository which is covered by\n"+
-        "https://www.alfresco.com/legal/agreements and https://www.alfresco.com/terms-use\n" +
-        "\n" +
+    String LICENCE =
         "License rights for this program may be obtained from Alfresco Software, Ltd. pursuant to a written agreement\n" +
         "and any use of this program without such an agreement is prohibited.\n" +
         "\n" ;
```
Comment (CezarLeahu) [inline alfresco-transformer-base/src/main/java/org/alfresco/transformer/logging/StandardMessages.java]: We may wish to change the text of the **`LICENCE`** constant to reference _LGPLv3_?
@montgolfiere

Comment (montgolfiere) [reply alfresco-transformer-base/src/main/java/org/alfresco/transformer/logging/StandardMessages.java]: Yes, can we find any example log startup text from other core ACS Community components - Repo, Search, Share, ... ?

Comment (tzclucian) [reply alfresco-transformer-base/src/main/java/org/alfresco/transformer/logging/StandardMessages.java]: Can't find anything similar to this around the mentioned repos so I'll proceed merging.
If we'll manage to find something we can easily update the `String`.

**Pull request merged**